moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -165,6 +163,47 @@ class Robot
                     const unsigned int num_steps = 0,
                     const std::string & point_name = std::string());
 
+
+  /**
+   * @brief addTrajPoint - add point to trajectory.
+   * @brief This function supports custom tool frame for adding traj points.
+   * @param traj_name - name of trajectory buffer to add point to
+   * @param point_name - name of point to add
+   * @param custom_tool_frame - frame (must be a TF accessible frame) in which pose is defined
```

Comment:
Change this argument to `tool_name`.  Then add an issue for expanding the source of the `tool_name` transforms (i.e. just like a point can be taken from multiple sources.